pypeit/tests/test_save.py

```diff
@@ -44,7 +44,7 @@ def test_save2d_fits():
     fitstbl['filename'] = 'b1.fits.gz'
     # Settings
     #settings.argflag['run']['directory']['science'] = data_path('')
-    setup = 'A_01_aa'
+    mastery_key = 'A_1_01'
```

Comment:
I assume you intended this "typo" because it's repeated? :smile:

Reply:
Hah!